src/main/kotlin/com/cognifide/gradle/aem/pkg/ComposeTask.kt

```diff
@@ -176,8 +180,12 @@ open class ComposeTask : Zip(), AemTask {
     fun includeBundles(project: Project) {
         dependProject(project)
 
+        val extraProperties = project.extensions.extraProperties
+        val bundlePath: String = if (extraProperties == null || !extraProperties.has("bundlePath")) config.bundlePath
+        else extraProperties.get("bundlePath") as String
+
         bundleCollectors += {
-            JarCollector(project).all.toList()
+            Pair(bundlePath, JarCollector(project).all.toList())
```

Comment:
Ok, but where does the parameter bundlePath come from? I mean a bundle module does not have an aemCompose task or config.